src/midiEditorCommands.cpp

```diff
@@ -436,11 +522,11 @@ pair<int, int> findChord(MediaItem_Take* take, int direction) {
 		return {-1, -1};
 	}
 	// Find the last note of the chord.
-	double firstStart = *firstNote;
+	double firstStart = (*firstNote).start;
```

Comment:
You should be able to do `firstNote->start` instead of `(*firstNote).start`. There are a few other instances of this too.

Reply:
So I did this, but that requires the -> operator to be implemented on the iterator. I did this now, but that's not going to work with temporary objects I'm afraid. Unless there's a workaround I do not know of.